src/main/java/appeng/client/render/cablebus/CableBusBakedModel.java

```diff
@@ -119,7 +120,15 @@
 						throw new IllegalStateException( "Trying to use an unregistered part model: " + model );
 					}
 
-					List<BakedQuad> partQuads = bakedModel.getQuads( state, null, rand );
+					List<BakedQuad> partQuads;
+					if( bakedModel instanceof IPartBakedModel )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS134)